gulpfile.js

```diff
@@ -86,7 +86,8 @@ gulp.task('shards', () => {
             'elements/kw-view-projects/kw-view-projects.html',
             'elements/kw-view-start/kw-view-start.html',
             'elements/kw-auth-modal/kw-auth-modal.html',
-            'elements/kw-blank-project-modal/kw-blank-project-modal.html'
+            'elements/kw-blank-project-modal/kw-blank-project-modal.html',
+            'elements/kw-alert-modal/kw-alert-modal.html'
```

Comment:
No need for that anymore. Actually the shard task doesn't need any `endpoints` defined as it gets them from `lazy-import` `link`s